helpers/deploy/deploymentUtils.ts

```diff
@@ -14,20 +15,48 @@ export const getAcmAdminAccount = async (network: SUPPORTED_NETWORKS): Promise<s
     return "0x285960C5B22fD66A736C7136967A3eB15e93CC67"; // ETHEREUM MULTISIG
   } else if (network === "opbnbtestnet") {
     return "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf"; // OPBNBTESTNET MULTISIG
+  } else if (network === "opbnbmainnet") {
+    return "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207"; // OPBNBMAINNET MULTISIG
   }
-
-  const normalTimelock = await ethers.getContract("NormalTimeLock");
+  const normalTimelock = await ethers.getContract("NormalTimelock");
   return normalTimelock.address;
 };
 
+export const getOmnichainGovernanceExecutorAdminAccount = async (network: SUPPORTED_NETWORKS): Promise<string> => {
+  const { deployer } = await getNamedAccounts();
+  if (network === "hardhat") {
+    return deployer;
+  } else if (network === "sepolia") {
+    return "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb"; // SEPOLIA MULTISIG
+  } else if (network === "ethereum") {
+    return "0x285960C5B22fD66A736C7136967A3eB15e93CC67"; // ETHEREUM MULTISIG
+  } else if (network === "opbnbtestnet") {
+    return "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf"; // OPBNBTESTNET MULTISIG
+  } else if (network === "opbnbmainnet") {
+    return "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207"; // OPBNBMAINNET MULTISIG
+  }
+  return deployer;
+};
+
 export const getOmnichainProposalSender = async (network: Omit<SUPPORTED_NETWORKS, "ethereum" | "sepolia">) => {
   if (network === "hardhat") {
     const omnichainProposalSenderAddress = (await ethers.getContract("OmnichainProposalSender")).address;
     return omnichainProposalSenderAddress;
-  } else if (network === "sepolia") {
+  } else if (network === "sepolia" || network === "opbnbtestnet") {
```

Comment:
This logic doesn't seem right because on sepolia and opbnbnbtestnet we are returning a bsc address. Rather than return an address we can use ethers to fetch the deployment

Reply:
We are fetching address of contract `OmnichainProposalSender` which is deployed on other network i.e. BSC. Hence we can't fetch directly.